dxfeed/wrappers/endpoint.py

```diff
@@ -1,5 +1,6 @@
 from dxfeed.core.DXFeedPy import *
 from dxfeed.wrappers.subscription import Subscription
+from datetime import datetime
 
 
 class Endpoint(object):
```

Comment:
Seems that part of the classes docstring should be moved to `__init__` method docstring